source/MaterialXView/RenderPipelineGL.cpp

```diff
@@ -370,6 +370,9 @@ void GLRenderPipeline::renderFrame(void*, int shadowMapSize, const char* dirLigh
                 float longitudeOffset = (lightRotation / 360.0f) + 0.5f;
                 envMaterial->modifyUniform("longitude/in2", mx::Value::createValue(longitudeOffset));
 
+                // Apply light intensity to the environment shader.
```

Comment:
Do we need a corresponding line to be added to RenderPipelineMetal.mm as well?

https://github.com/AcademySoftwareFoundation/MaterialX/blob/main/source/MaterialXView/RenderPipelineMetal.mm#L491

Reply:
<img width="936" alt="image" src="https://github.com/AcademySoftwareFoundation/MaterialX/assets/1727158/585d02ae-713a-4788-9105-96b083f7cea7">
Thanks @jstone-lucasfilm, I had missed this. I've updated it now, and verified it on MacOS.